### ISSUE TITLE
About adding KGCN as an example of Recommender Systems

### ISSUE THREAD

**Issue by pengfei** [dmlc/dgl#1138]
## 🚀 Feature
Can kgcn be used as an example of a recommendation system? 
Examples of recommendation systems are few. 
KGCN involves neighbor sampling, which is a good example of Nodeflow.
It's also not too complicated

(www19)Knowledge Graph Convolutional Networks for Recommender Systems
[https://arxiv.org/abs/1904.12575](url)
[https://github.com/hwwang55/KGCN](url)

**Comment by zzhang-cn** [comment]
I agree. Would you like to take a lead on it and we can assign someone to help?

**Comment by pengfei** [comment]
I am a beginner of DGL, and it may be more suitable for others who are more familiar with DGL to implement.

**Comment by jermainewang** [comment]
Currently, we are lacking the bandwidth on the whole DGL+Recsys direction. Labeled this with help wanted.
